models/yang/sonic/sonic-sflow.yang

```diff
@@ -86,8 +93,9 @@ module sonic-sflow {
                     }
 
                     leaf agent_id {
-                       sonic-ext:custom-validation ValidateSflowAgentId;
-                       type string;
+                        type leafref {
```

Comment:
Why are we deleting the agend_id validation now? I think you meant to undo your changes in favor of the existing code...